<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>openembedded-core - OpenEmbedded Core layer</title>
<meta name='generator' content='cgit v0.9.2-21-gd62e'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://cgit.openembedded.org/openembedded-core/atom/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/'><img src='/oe.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/'>index</a> : <a title='openembedded-core' href='/openembedded-core/'>openembedded-core</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='2011-1'>2011-1</option>
<option value='daisy'>daisy</option>
<option value='danny'>danny</option>
<option value='denzil'>denzil</option>
<option value='dizzy'>dizzy</option>
<option value='dora'>dora</option>
<option value='dylan'>dylan</option>
<option value='master' selected='selected'>master</option>
<option value='master-next'>master-next</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>OpenEmbedded Core layer</td><td class='sub right'>OpenEmbedded</td></tr></table>
<table class='tabs'><tr><td>
<a href='/openembedded-core/'>summary</a><a href='/openembedded-core/refs/'>refs</a><a href='/openembedded-core/log/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch'>log</a><a class='active' href='/openembedded-core/tree/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch'>tree</a><a href='/openembedded-core/commit/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch'>commit</a><a href='/openembedded-core/diff/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch'>diff</a><a href='/openembedded-core/about/'>about</a></td><td class='form'><form class='right' method='get' action='/openembedded-core/log/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/openembedded-core/tree/'>root</a>/<a href='/openembedded-core/tree/meta'>meta</a>/<a href='/openembedded-core/tree/meta/recipes-devtools'>recipes-devtools</a>/<a href='/openembedded-core/tree/meta/recipes-devtools/rpm'>rpm</a>/<a href='/openembedded-core/tree/meta/recipes-devtools/rpm/rpm-4.11.2'>rpm-4.11.2</a>/<a href='/openembedded-core/tree/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch'>fix_libdir.patch</a></div><div class='content'>blob: be0626c8b30163f9a941394afaddf84e3267179d (<a href='/openembedded-core/plain/meta/recipes-devtools/rpm/rpm-4.11.2/fix_libdir.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a class='no' id='n1' name='n1' href='#n1'>1</a>
<a class='no' id='n2' name='n2' href='#n2'>2</a>
<a class='no' id='n3' name='n3' href='#n3'>3</a>
<a class='no' id='n4' name='n4' href='#n4'>4</a>
<a class='no' id='n5' name='n5' href='#n5'>5</a>
<a class='no' id='n6' name='n6' href='#n6'>6</a>
<a class='no' id='n7' name='n7' href='#n7'>7</a>
<a class='no' id='n8' name='n8' href='#n8'>8</a>
<a class='no' id='n9' name='n9' href='#n9'>9</a>
<a class='no' id='n10' name='n10' href='#n10'>10</a>
<a class='no' id='n11' name='n11' href='#n11'>11</a>
<a class='no' id='n12' name='n12' href='#n12'>12</a>
<a class='no' id='n13' name='n13' href='#n13'>13</a>
<a class='no' id='n14' name='n14' href='#n14'>14</a>
<a class='no' id='n15' name='n15' href='#n15'>15</a>
<a class='no' id='n16' name='n16' href='#n16'>16</a>
<a class='no' id='n17' name='n17' href='#n17'>17</a>
<a class='no' id='n18' name='n18' href='#n18'>18</a>
<a class='no' id='n19' name='n19' href='#n19'>19</a>
</pre></td>
<td class='lines'><pre><code>Upstream-Status: Inappropriate [OE-Core specific]

Signed-off-by: Saul Wold &lt;sgw&#64;linux.intel.com&gt;
Signed-off-by: Ronan Le Martret &lt;ronan.lemartret&#64;open.eurogiciel.org&gt;


diff --git a/installplatform b/installplatform
index 8c3eba0..fa15e91 100755
<span class="hl kwb">--- a/installplatform</span>
<span class="hl kwa">+++ b/installplatform</span>
&#64;&#64; -112,7 +112,7 &#64;&#64; for ARCH in noarch `grep ^arch_canon $RPMRC | cut -d: -f2`; do
   [ -z &quot;$CANONARCH&quot; ] &amp;&amp; continue
 
   if [ &quot;$OS&quot; = &quot;linux&quot; ] &amp;&amp; [ &quot;$CANONCOLOR&quot; = 3 ]; then
<span class="hl kwb">-      LIB=${LIB}64</span>
<span class="hl kwa">+      LIB=${LIB}</span>
   fi
 
   PPD=&quot;${DESTDIR}/${platformdir}/${ARCH}-${OS}&quot;
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.9.2-21-gd62e at 2015-01-26 10:44:42 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
